testing/jormungandr-integration-tests/src/jormungandr/grpc/client_tests.rs

```diff
@@ -396,3 +396,124 @@ pub fn pull_blocks_hashes_wrong_order() {
 
     assert!(result.is_err());
 }
+
+#[test]
+pub fn test_watch_block_subscription_blocks_are_in_logs() {
+    use std::collections::HashSet;
+
+    let setup = setup::client::default();
+
+    let watch_client = setup.watch_client;
+
+    let (sender, receiver) = std::sync::mpsc::channel();
+
+    watch_client.block_subscription(sender);
```

Comment:
we've already discusses about this but I find it quite confusing 😅

Reply:
I don't understand, what is it that we discussed?

Reply:
the naming, I find it a bit confusing that this is a client and is on the sender part of the stream

Reply:
oh, well... But in this case this is actually a client (the grpc client). It's on the sender part of the stream only because this coe is sync, so this just loops on a separate threads and sends the messages on that channel, but the `watch_client` is a client for the node.